8/jdk/al2022/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM amazonlinux:2022
 
-ARG version=1.8.0_352.b08-1
+ARG version=1.8.0_362.b08-1
 
 
 RUN set -eux \
```

Comment:
resource_version needs to get updated. There was an extra ".0" in the version last release that has been fixed.